src/transformers/training_args.py

```diff
@@ -870,11 +893,38 @@ class TrainingArguments:
         default=0,
         metadata={
             "help": (
-                "FSDP's minimum number of parameters for Default Auto Wrapping. (useful only when `fsdp` field is"
-                " passed)."
+                "This parameter is deprecatetd. FSDP's minimum number of parameters for Default Auto Wrapping. (useful"
+                " only when `fsdp` field is passed)."
             )
         },
     )
+    fsdp_config: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": (
+                "Config to be used with FSDP (Pytorch Fully Sharded  Data Parallel). The  value is either the of "
```

Comment:
```suggestion
                "Config to be used with FSDP (Pytorch Fully Sharded  Data Parallel). The  value is either the of
```suggestion
                "Config to be used with FSDP (Pytorch Fully Sharded  Data Parallel). The  value is either a "
```
 "
```

Reply:
Done. Can we merge it now ?